File: Domain/Patient.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Domain
{
    public class Patient
    {
        [Key]
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Pesel { get; set; }
        public DateTime BirthDate { get; set; }
        public Gender Gender { get; set; }
        public virtual ICollection<Appointment> Appointments { get; set; }
    }

}

File: client-app/src/app/models/patient.ts
import { IAppointment } from "./appointment";

export interface IPatient {
  id: number;
  firstName: string;
  lastName: string;
  birthDate: Date;
  pesel: string;
  gender: string;
  appointments: IAppointment[];
}

export interface IPatientFormValues {
  id?: number
  firstName: string;
  lastName: string;
  pesel: string;
  birthDate?: Date;
  gender: string;
}
